plugin/cache.go

```diff
@@ -88,13 +88,33 @@ func allOpKeysIncludingChildrenRegex(path string) *regexp.Regexp {
 }
 
 // ClearCacheFor removes entries from the cache that match or are children of the provided path.
-// If successful, returns an array of deleted keys.
+// If successful, returns an array of deleted keys. Optionally clear the list operation for the
+// parent to remove any attributes related to the specified entry.
 //
 // TODO: If path == "/", we could optimize this by calling cache.Flush(). Not important
 // right now, but may be worth considering in the future.
-func ClearCacheFor(path string) []string {
+func ClearCacheFor(path string, clearParent bool) []string {
```

Comment:
Nit: `clearParent` => `clearParentList` (might as well document that in the param name)

Reply:
It might be nice to document that `clearParent == false` would be useful if the caller plans on modifying the cached entry map instead.

Reply:
Modifying the cached entry map is only possible within the plugin module. So probably not useful to document generally.

Reply:
Hmm, if that's the case, when would `clearParent == false` be useful to people outside the plugin package? Maybe `ClearCacheFor` could remain as-is, and we just have an internal `clearCacheFor` wrapper that does the real work? Not a blocker, just a thought.

Reply:
Nevermind, I see it used in the `volume` package. So I guess for testing it makes it so you don't have to mock the parent's cache. Still feels weird.

Reply:
There might be other cases, but it does feel fairly special-case. Maybe we remove it later.